styleguide/screens/Patterns/Modals/Modals.js

```diff
@@ -0,0 +1,279 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import dedent from 'dedent';
+
+import Specimen from '../../../components/Specimen/Specimen';
+import { D, H, T, C, Note, Placeholder } from '../../../components/Scaffold/Scaffold';
+import ModalAnimator from '../../../../components/Modal/ModalAnimator';
+import Window, { WindowTitle } from '../../../../components/Modal/Window';
+import WithCross from '../../../../components/Modal/WithCross';
+import Btn from '../../../../components/Btn/Btn';
+
+import shared from '../../../../styleguide/shared.css';
+import css from './Modals.css';
+import m from '../../../../globals/modifiers.css';
+
+export default class ModalDocumentation extends Component {
+  state = {
+    modalAnimator: false,
+    modalWindow: false,
+    modalWindowDark: false,
+    modalWithCross: false,
+    modalWithCrossDark: false,
+  };
+
+  toggleModalAnimator = () => {
+    this.setState(({ modalAnimator }) => ({
+      modalAnimator: !modalAnimator,
+    }));
+  };
+
+  toggleModalWindow = () => {
+    this.setState(({ modalWindow }) => ({
+      modalWindow: !modalWindow,
+    }));
+  };
+
+  toggleModalWindowDark = () => {
+    this.setState(({ modalWindowDark }) => ({
+      modalWindowDark: !modalWindowDark,
+    }));
+  };
+
+  toggleModalWithCross = () => {
+    this.setState(({ modalWithCross }) => ({
+      modalWithCross: !modalWithCross,
+    }));
+  };
+
+  toggleModalWithCrossDark = () => {
+    this.setState(({ modalWithCrossDark }) => ({
+      modalWithCrossDark: !modalWithCrossDark,
+    }));
+  };
+
+  render() {
+    return (
+      <div>
+        <H level={ 1 }>Modals</H>
+        <T elm="p" className={ cx(m.mtr, m.largeI, m.demi) }>
+          Modals are a secondary window that allow you to display additional
```

Comment:
Not sure if it worth mentioning that we use portals for our modals only because it's not an approach that I have seen before, but I understand if it is not relevant because it's not really a design decision.